addons/power/functions/fnc_initFuelLevelWithEdenAttribute.sqf

```diff
@@ -14,22 +15,24 @@ params ["_generator"];
 {
     params ["_generator"];
 
-    // wait until the eden attribute is set via expression; unset is -1; regular values should be between 0 and 1
-    waitUntil { !(isNil {_generator getVariable "AE3_EdenAttribute_FuelLevel"}) };
-
-    private _edenAttributeFuelLevel = _generator getVariable "AE3_EdenAttribute_FuelLevel";
-
     // wait until all "init" processes are done, see: https://community.bistudio.com/wiki/Initialization_Order
     waitUntil { !isNil "BIS_fnc_init" };
 
-    // only set power level if changed in Eden Editor; -1 is default value
-    if (_edenAttributeFuelLevel != -1) then
+    private _edenAttributeFuelLevel = _generator getVariable ["AE3_EdenAttribute_FuelLevel", nil];
```

Comment:
Default value unnecessary. If the variable is undefined, it will be nil anyways

Reply:
Yes, that's true. But I wanted to train myself to always take care of default values.